clients/fides-js/src/lib/consent-utils.ts

```diff
@@ -29,14 +29,26 @@ export const debugLog = (
 };
 
 /**
- * Returns true if privacy experience is null or empty
+ * Returns true if the provided input is a valid PrivacyExperience object.
+ *
+ * This includes the special case where the input is an empty object ({}), which
+ * is a valid response when the API does not find a PrivacyExperience configured
+ * for the given geolocation.
  */
 export const isPrivacyExperience = (
   obj: PrivacyExperience | undefined | EmptyExperience
 ): obj is PrivacyExperience => {
```

Comment:
I happened to stumble on this PR and this change is a little dangerous type-wise—we claim that returning true means the obj passed in was of type `PrivacyExperience`, but this change makes it so it can be either `PrivacyExperience` or `EmptyExperience`. the type predicate will still say it's `PrivacyExperience` though!